Platform/ApollolakeBoardPkg/CfgData/Template_CfgData.dsc

```diff
@@ -58,7 +58,30 @@
 
       # !BSF NAME:{Pid Pad Info}
       # !BSF TYPE:{Combo}
-      # !BSF OPTION:{0x0F0C4:GPIO_PMIC_STDBY, 0x0D0C4:GPIO_213, 0x0C8C4:GPIO_PMIC_RESET_B, 0x0C0C4:GPIO_PMIC_PWRGOOD}
+      # !BSF OPTION:{ 0x000C5:GPIO_0, 0x008C5:GPIO_1, 0x010C5:GPIO_2, 0x018C5:GPIO_3, 0x020C5:GPIO_4, 0x028C5:GPIO_5, 0x030C5:GPIO_6, 0x038C5:GPIO_7, }
```

Comment:
It looks like every options are the same as PinOption's one.  Can we make a single one and re-use it? Like using include GpioOption file? Or better idea?

Reply:
Agree with Aiden. should use "!include CfgData_GpioPinOption.dsc" here. 
but it looks there is other bug to support include file in template. so we could use include after tool issue is fixed.